storybookjs/storybook
# manage access to localStorage\sessionStorage with store2 pkg

Issue: In browsers that deny access to localStorage \ sessionStorage storybook would fail to load.

## What I did
Changed the way `storybook` access localStorage \ sessionStorage to be managed with `store2` package. store2 has a fallback when there is no access to localStorage \ sessionStorage it'll create a in-memory storage.

## How to test

- Is this testable with Jest or Chromatic screenshots?
 -> No (as far as i know)
- Does this need a new example in the kitchen sink apps?
 -> No
- Does this need an update to the documentation?
 -> No

If your answer is yes to any of these, please make sure to include it in your PR.
